demo/app.component.ts

```diff
@@ -85,60 +93,66 @@ export class AppComponent implements OnInit {
     children: [
       {
         value: 'Serif  -  All my children and I are STATIC ¯\\_(ツ)_/¯',
+        id: 1,
         settings: {
           'static': true
         },
         children: [
-          { value: '<a href="#" id="antiqua" class="test">Antiqua</a> with HTML tags.'},
```

Comment:
Oy, I didn't notice that I removed this, I will rebase again and will keep this. Sorry, it wan't intentional.